src/wp-includes/version.php

```diff
@@ -37,7 +37,7 @@
  *
  * @global string $required_php_version
  */
-$required_php_version = '5.6.20';
+$required_php_version = '7.0.0';
```

Comment:
Shouldn't we set `7.0.33` (released on 6 Dec 2018) as a minimum?

Reply:
Thanks for the suggestion Jb. When the last minimum version bump was made (see [#46594](https://core.trac.wordpress.org/ticket/46594)) the minimum version was originally set to 5.6.30 but this got reduced to 5.6.20 after concerns around common tooling that uses older patch versions and unnecessarily excluding sites that are running older patch versions.

There's no major technical advantage to WordPress core using a later patch version in the 7.0 branch as the minimum, the only one mentioned on [#57345](https://core.trac.wordpress.org/ticket/57345) so far is 7.0.10 for the `IMG_WEBP` constant which is a minor advantage. We're going to go with 7.0.0.